memorious/operations/aleph.py

```diff
@@ -132,6 +136,57 @@ def aleph_folder(context, data):
             backoff(ae, try_number)
 
 
+def aleph_entity(context, data):
+    api = get_api(context)
+    if api is None:
+        return
+    collection_id = get_collection_id(context, api)
+    content_hash = data.get("content_hash")
+    source_url = data.get("source_url", data.get("url"))
+    foreign_id = data.get("foreign_id", data.get("request_id", source_url))
+    # Fetch document id from cache
+    document_id = context.get_tag(make_key(collection_id, foreign_id, content_hash))
+    if document_id:
+        context.log.info("Skip aleph upload: %s", foreign_id)
+        data["aleph_id"] = document_id
+        context.emit(data=data, optional=True)
+        return
+    else:
+        document_id = uuid.uuid4().hex
+
+    for try_number in range(api.retries):
+        rate = settings.MEMORIOUS_RATE_LIMIT
+        rate_limit = get_rate_limit("aleph", limit=rate)
+        rate_limit.comply()
+
+        try:
+            api.write_entities(
```

Comment:
`api.write_entities` uses the bulk api. That means entities uploaded through it won't go through the NER analysis pipeline. That kind of defeats the purpose, especially when uploading Article entities through it.

I think we should add an `api.write_entity` method in alephclient to upload a single entity through the `POST` endpoint and use that here. That will ensure uploaded Articles get NERed.